presets/symfony/preset-config.yml

```diff
@@ -0,0 +1,19 @@
+language: php
+commands:
+  create:
+  - kool docker kooldev/php:7.4 composer create-project --prefer-dist symfony/website-skeleton
+questions:
+  database:
+    message: What database service do you want to use
+    options:
+      mysql8: MySQL 8.0
```

Comment:
for the sake of clarity and future extendability maybe we might change the format of this `options:` entry from the current object to an array like:

```yml
  options:
    - name: MySQL 8.0
      template: mysql8
    - name: PostgreSQL 13
      template: postgresql13
```

We could do this in a future iteration as well - the goal being to  make clear the importance of the key (template name) on these options setting. Having an array of objects like this could possibly help I guess.